--- alsa-tools-1.0.28/as10k1/as10k1.c.omv~	2014-08-01 21:26:31.572896504 +0200
+++ alsa-tools-1.0.28/as10k1/as10k1.c	2014-08-01 21:26:47.716344683 +0200
@@ -366,7 +366,7 @@ void as_exit(const char *message)
 	exit(1);
 }
 
-inline void output_tram_line( struct list_head *line_head, int type){
+static void output_tram_line( struct list_head *line_head, int type){
         
         struct tram *tram_sym;
         struct list_head *entry;
