--- ./sscape_ctl/sscape_ctl.c.orig	2008-10-29 13:42:12.000000000 +0100
+++ ./sscape_ctl/sscape_ctl.c	2008-12-14 09:42:49.000000000 +0100
@@ -34,7 +34,7 @@
 
 const char default_dir[] = "/sndscape";
 const char scope[] = "scope.cod";
-unsigned char _microcode[SSCAPE_MICROCODE_SIZE];
+
 
 static void
 show_usage(void)
@@ -142,7 +142,7 @@
   {
     int save_errno;
 
-    err = read(fd, microcode->code, sizeof(_microcode));
+    err = read(fd, microcode->code, sizeof(microcode->code));
     if (err >= 0)
     {
       printf("Microcode: read %d bytes\n", err);
@@ -225,7 +225,6 @@
     struct sscape_bootblock  boot;
     struct sscape_microcode  microcode;
 
-    microcode.code = _microcode;
     if ((len = get_directory(directory, filename, sizeof(filename))) == 0)
     {
       fprintf(stderr, "Invalid directory - pathname too long\n");
